README.md

```diff
@@ -1,133 +1,45 @@
+![](./docs/img/QMlogoBig.png)
 
-![](./QMigrate/Figures/Title.png)
+QuakeMigrate is a Python package for the detection and location of seismicity from passive seismic data using waveform migration and stacking. It can be used to produce catalogues of earthquakes, including hypocentres, origin times, phase arrival picks, and local magnitude estimates, as well as rigorous estimates of the associated uncertainties.
 
-> QuakeMigrate is a Python and C software package that detects and locates seismic events from raw data, using a migration and coalescence back projection technique. The ability to fine tune the input parameters and analyse the results using a suite of visualisation functions give it an edge over the alternatives. Simple scripts (see the provided examples) allow the user to rapidly deploy the software in any seismological setting. The open source and module nature allows for the continious development with the inclusion of cutting edge techniques. 
+The package has been implemented with a modular architecture with the potential for extension and adaptation at numerous entry points, including the calculation of traveltime grids, the choice of characteristic function, and the algorithm used to perform phase picking.
 
----
-<img src="./QMigrate/Figures/References.png" alt="drawing" height="50"/>
+Documentation
+-------------
+Documentation for QuakeMigrate is hosted [here](https://quakemigrate.readthedocs.io/en/development/index.html).
 
-QuakeMigrate is currently in prperation for publication. Once accepted we hope that the publication will have the reference: 
-  
-  `Smith, J.D., Winder, T., Hudson, T.S., Bacon, C., Greenfield, T., Drew, J. and R.S. White (2019), QuakeMigrate: Earthquake Onset Detection adn Back-Migration Location Method, Seismological Research Letters.`
+Installation
```

Comment:
Not sure this is the done thing, but at least for the time being maybe worth shouting about the platform independence here?